freshmaker/lightblue.py

```diff
@@ -581,9 +581,9 @@ def _make_request(self, entity, data):
                 record_mode=conf.vcrpy_mode,
             )
             with my_vcr.use_cassette(f'{self.event_id}.yml'):
-                response = requests.post(entity_url, **request_kwargs)
+                response = requests.post(entity_url, **request_kwargs, timeout=conf.requests_timeout)
         else:
-            response = requests.post(entity_url, **request_kwargs)
+            response = requests.post(entity_url, **request_kwargs, timeout=300)
```

Comment:
Not sure if 300s is good enough, we can have another timeout variable in conf for lightblue requests, so we can easily adjust it if we see any issues, or maybe:
```timeout=max(600, conf.requests_timeout*5)```

Reply:
@qixiang Fixed it, you can take a look